index.js

```diff
@@ -0,0 +1,39 @@
+const yaml = require('js-yaml');
+
+module.exports = robot => {
+    robot.on('pull_request.opened', async context => {
+        const owner = context.payload.repository.owner.login;
+        const repo = context.payload.repository.name;
+        const number = context.payload.pull_request.number;
```

Comment:
you can use `context.repo()` instead of sperate repo and name variables.`contex. repo()` returns {repo, name} already :)

docs: https://probot.github.io/probot/latest/Context.html#repo